src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -446,7 +446,7 @@ public static object MarkAsSecurityArtifact(object arg, Func<object, string> cal
             if (!_isHeaderWritten)
             {
                 string headerMessage = string.Format(CultureInfo.InvariantCulture, "Microsoft.IdentityModel Version: {0}. Date {1}. {2}",
-                    typeof(IdentityModelEventSource).Assembly!.GetName().Version!.ToString(),
+                    typeof(IdentityModelEventSource).Assembly.GetName().Version?.ToString(),
```

Comment:
there should never be the case where Version is null.

Reply:
true but `GetName()` can return null.